lessons/pydata/eda-univariate-timeseries/index.ipynb

```diff
@@ -0,0 +1,4745 @@
+{
```

Comment:
<p>knihovnu physt</p><p> </p><p>Ale jinak "na vlastní nebezpečí" je na místě :-)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='97'/>

Reply:
<p>Dal jsem to tam takhle schválně, ale jestli to to zní příliš nemístně, tak to poupravím :-D </p><p> </p>